independent-publisher-connectors/AzureFunctionApps/apiDefenition.swagger.json

```diff
@@ -0,0 +1,89 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "Default title",
+        "description": "",
+        "version": "1.0"
+    },
+    "host": "[YOURFUNCTIONAPP].azurewebsites.net",
+    "basePath": "/",
+    "schemes": [
+        "https"
+    ],
+    "consumes": [],
+    "produces": [],
+    "paths": {
+        "/api/DynamicsAuthStart": {},
```

Comment:
@Stephan-Rijk can you please explain the empty paths? is it possible to define them or remove them?